pages/index.vue

```
@@ -60,14 +77,136 @@ const cleanSearch = computed(() => {
   return getCleanText(search.value);
 });
 
+const requestLocationPermission = async () => {
+  if (!navigator.geolocation) {
+    ElMessage.error("Geolocalização não é suportada neste navegador.");
+    return;
+  }
+
+  try {
+    const position = await new Promise<GeolocationPosition>((resolve, reject) => {
```

ESLint

[error] Insert `⏎······`

(prettier/prettier)